lib/presentation/pages/login/login_view.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:get/get.dart';
+import 'package:task/app/app_colors.dart';
+import 'package:task/app/themes.dart';
+import 'package:task/presentation/pages/login/login_controller.dart';
+import 'package:task/presentation/pages/login/widgets/custom_form_field.dart';
+
+import 'widgets/custom_social_button.dart';
+
+class LoginScreen extends GetView<LoginController> {
+  const LoginScreen({super.key});
+  @override
+  Widget build(BuildContext context) {
+    //final controller = Get.put(LoginController());
+    final size = MediaQuery.of(context).size;
+    return SafeArea(
+        child: Scaffold(
+      body: SingleChildScrollView(
+          child: Obx(
+        () => Form(
+          key: controller.loginKey.value,
+          onChanged: () => controller.toggleLoginButton(),
+          autovalidateMode: AutovalidateMode.onUserInteraction,
+          child: Padding(
+            padding: const EdgeInsets.only(top: 8.0),
+            child: Column(children: [
+              Container(
+                decoration: const BoxDecoration(
+                    image: DecorationImage(
+                        image: AssetImage("assets/images/login.jpg"),
+                        fit: BoxFit.cover)),
+                height: size.height * 0.4,
+              ),
+              const SizedBox(
```

Comment:
Make a class for size box widget to refactor the code and pass value of height and width with optional parameters